tools/autodeployment/builds.js

```diff
@@ -112,9 +112,9 @@ module.exports.buildLogHandler = function handleLog(req, res, next) {
 
   res.writeHead(200, { 'Content-Type': 'text/plain' });
 
-  const onData = (data) => res.write(data);
-  const onError = () => end('Error, end of log.');
-  const onEnd = () => end('Build Complete.');
+  let onData;
+  let onError;
+  let onEnd;
```

Comment:
Isn't the old code more precise?

Reply:
The `end` function is used before its definition.  It works but ESLint recommends not to do that. 
https://eslint.org/docs/rules/no-use-before-define

Reply:
oh, that's right, I missed the `end` declaration.